packages/shopify-codemod/test/fixtures/flatten-iife-global-assignment/inconsistent-naming.input.js

```diff
@@ -0,0 +1,9 @@
+App.A = (function() {
+  function Q() {}
+
+  Q.prototype.foo = function() {
+    return 'bar';
+  };
+
```

Comment:
Now that you mention it, got another couple of test cases that I want to handle just in case, for eg:

```
App.A = (function() {
  function A() {}

  A.prototype.woof = function() {}

  A.factoryFunction = function() {
    return new A();
  }

  return A.factoryFunction;
})();
```